### ISSUE TITLE
Weird page returns on login

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#159]
I've been trying to track this problem down and today I was prepared and took screenshots.  When my time has expired and I have to login to the website, I get a page that looks like this:
![screen shot 2015-04-16 at 11 24 01 am](https://cloud.githubusercontent.com/assets/11475316/7184445/466b9496-e42b-11e4-8d1d-ccdf51840087.png)

When I see this particular page, it is an indication of doom, because I log in, and then this page pops up that never returns to PATRIC
![screen shot 2015-04-16 at 11 24 08 am](https://cloud.githubusercontent.com/assets/11475316/7184458/5fdace10-e42b-11e4-8c14-fc813215351b.png)

**Comment by mshukla1** [comment]
Rebecca, do you still see this problem?

**Comment by ARWattam** [comment]
I'll look for it.  It happens after an odd series that I can describe if anyone is interested.

**Comment by ARWattam** [comment]
It just happened again.  My browser had been idle all night, and it appeared like I was logged in.  I started using the site, and it still looked like I was logged in.  I tried to access my workspace, and it asked me to log in.  This is what it looks like when that happens, and then I know that the same old problem is going to happen
![screen shot 2015-06-12 at 10 07 11 am](https://cloud.githubusercontent.com/assets/11475316/8131900/8a354838-10eb-11e5-9c6f-2174e12f7455.png)


Then, instead of showing my  workspace, or anything PATRIC, this page loads.

![screen shot 2015-06-12 at 10 07 26 am](https://cloud.githubusercontent.com/assets/11475316/8131928/b96c0a6a-10eb-11e5-913d-ddcc9171cf54.png)

This is the url at the top of the blank page
![screen shot 2015-06-12 at 10 07 28 am](https://cloud.githubusercontent.com/assets/11475316/8131932/c36759ca-10eb-11e5-9263-5ed52184eb6e.png)
